foundation_cms/static/scss/components/primary_nav.scss

```diff
@@ -5,6 +5,8 @@ $hamburger-bar-spacing: 0.8rem;
 @import "../../../../node_modules/delicious-hamburgers/scss/base";
 @import "../../../../node_modules/delicious-hamburgers/scss/animations/collapse";
 
+$primary-nav-height: rem-calc(88);
```

Comment:
If `$hamburger-bar-spacing` and `$primary-nav-height` aren't used by other files. Let's move them to within `.primary-nav-ns { ... }` to avoid making them global.